### ISSUE TITLE
Implement Issue List with GitHub Primer Components

### ISSUE THREAD

**Issue by megamanics** [builtwithai/ui#2]
The Issue list should be refactored to use GitHub Primer components. This would provide a more consistent look and feel with other GitHub interfaces, and potentially offer better accessibility and responsiveness.
